includes/template-library/sources/local.php

```diff
@@ -1357,6 +1358,17 @@ public function print_blank_state_template( $current_type_label, $href, $descrip
 					printf( __( 'Add New %s', 'elementor' ), $current_type_label );
 					?>
 				</a>
+				<?php
+				$module_name = str_replace( '_', ' ', module::MODULE_NAME );
```

Comment:
Never use labels, they may be translated. try to find the type via global `$typenow` or via `get_current_screen()`.

Reply:
Fixed